src/lib/swapper/swappers/ThorchainSwapper/utils/getTradeRate/getTradeRate.ts

```diff
@@ -86,7 +88,7 @@ export const getTradeRate = async ({
   )
 
   const { data } = await thorService.get<ThornodeQuoteResponse>(
-    `${deps.daemonUrl}/lcd/thorchain/quote/swap?amount=${sellAmountCryptoThorBaseUnit}&from_asset=${sellPoolId}&to_asset=${buyPoolId}&destination=${receiveAddress}`,
+    `${deps.daemonUrl}/lcd/thorchain/quote/swap?amount=${sellAmountCryptoThorBaseUnit}&from_asset=${sellPoolId}&to_asset=${buyPoolId}&destination=${receiveAddress}&affiliate_bps=${affiliateBps}&affiliate=ss`,
```

Comment:
```suggestion
    `${deps.daemonUrl}/lcd/thorchain/quote/swap?amount=${sellAmountCryptoThorBaseUnit}&from_asset=${sellPoolId}&to_asset=${buyPoolId}&destination=${receiveAddress}&affiliate_bps=${affiliateBps}&affiliate=ss`,
```

template `ss` in from `THORCHAIN_AFFILIATE_NAME`?

Reply:
Good call. Tackled downstack in https://github.com/shapeshift/web/pull/4361/commits/fdd6eaadcb83f2708e845d022d0eedf8eb66ef2c.

Reply:
and https://github.com/shapeshift/web/pull/4353/commits/886e7cd5c8727e4ab88c09019a750b8907790613